### DISCUSSION TITLE
After Install, just flashes and then nothing

### DISCUSSION THREAD

**Discussion by JayS-PA** [ImageMagick/ImageMagick#2130] [Help]
All,

Have tried to install different version on Win 10 BootCamp partition running on Macbook Pro.  The install runs successfully, but when I try to launch ImageMagick, I see a flash (like a UI is trying to launch) then nothing.  Have tried 32 and 64 bit, installed the latest C++ environment as well.  Appreciate any assistance.  I'm trying to convert a B&W image to a color by modifying RGB values.

Jay S.

**Comment by dlemstra** [comment]
ImageMagick is a command line program without a UI. What is the command that are you running? And why do you want to run it on Windows on a Macbook?

**Reply by JayS-PA** [reply]
Thanks for replying.  Just trying the command string to test install and display the Wizard jpeg image.  As for running Windows on a Mac, it's very commonly done.  It's Intel based chip, so once I boot the Windows partition, it's like any other laptop.  It's a much better solution than virtualizing a Windows environment and running it under OSX.  Effectively a dual boot machine.
